### DISCUSSION TITLE
How to change sidebery language?

### DISCUSSION THREAD

**Discussion by theurichde** [mbnuqw/sidebery#817] [Q&A] [answered]
Hello!

I liked my sidebery v4 in English a lot, but installing v5 changed the UI to German. Is there any way to force the plugin to switch back to English, without having to change my complete browser language to English? Searching the plugin settings doesn't show a pick-a-language-dialog.

**Comment by mbnuqw** [comment, marked as answer]
No, there is no way to do this right now, but I can add it.
#818

**Reply by theurichde** [reply]
That would be great :-) - very much appreciated!

**Reply by bennygoldflesh** [reply]
Please add it man. Thank you for your work.

**Reply by tfriedel** [reply]
yeah, would be great!
